modules/migration-launching-cam.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_openshift_3_to_4/migrating-openshift-3-to-4.adoc
+[id="migration-launching-cam_{context}"]
+= Launching the CAM web console
+
+.Procedure
+
+. If you are using self-signed certificates, launch a browser and accept the CA certificates manually for the following:
```

Comment:
Do you have to try to navigate to the CAM web console before this step?

Reply:
Nope